From 022999fe6c4008d1ca6ec0f33d7f7c88db97f3fa Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Wed, 1 Nov 2017 11:11:30 +0530
Subject: [PATCH] staging: fsl_ppfe/eth: add pfe support to Kconfig and
 Makefile

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
[ Aisheng: fix minor conflict due to removed VBOXSF_FS ]
Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
---
 MAINTAINERS              | 8 ++++++++
 drivers/staging/Kconfig  | 1 +
 drivers/staging/Makefile | 1 +
 3 files changed, 10 insertions(+)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -6575,6 +6575,14 @@ F:	drivers/ptp/ptp_qoriq_debugfs.c
 F:	include/linux/fsl/ptp_qoriq.h
 F:	Documentation/devicetree/bindings/ptp/ptp-qoriq.txt
 
+FREESCALE QORIQ PPFE ETHERNET DRIVER
+M:	Anji Jagarlmudi <anji.jagarlmudi@nxp.com>
+M:	Calvin Johnson <calvin.johnson@nxp.com>
+L:	netdev@vger.kernel.org
+S:	Maintained
+F:	drivers/staging/fsl_ppfe
+F:	Documentation/devicetree/bindings/net/fsl_ppfe/pfe.txt
+
 FREESCALE QUAD SPI DRIVER
 M:	Han Xu <han.xu@nxp.com>
 L:	linux-spi@vger.kernel.org
--- a/drivers/staging/Kconfig
+++ b/drivers/staging/Kconfig
@@ -126,5 +126,6 @@ source "drivers/staging/exfat/Kconfig"
 source "drivers/staging/qlge/Kconfig"
 
 source "drivers/staging/fsl_qbman/Kconfig"
+source "drivers/staging/fsl_ppfe/Kconfig"
 
 endif # STAGING
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -54,3 +54,4 @@ obj-$(CONFIG_USB_WUSB)		+= wusbcore/
 obj-$(CONFIG_EXFAT_FS)		+= exfat/
 obj-$(CONFIG_QLGE)		+= qlge/
 obj-$(CONFIG_FSL_SDK_DPA)      += fsl_qbman/
+obj-$(CONFIG_FSL_PPFE)		+= fsl_ppfe/
